frontend/src/components/app/RawModal.js

```diff
@@ -357,12 +361,14 @@ class RawModal extends Component {
               </span>
               {!!rawModal.headerProperties && (
                 <div className="optional">
-                  {rawModal.headerProperties.entries.map((entry, idx) => (
-                    <span key={idx} className="optional-name">
-                      <p className="caption">{entry.caption}:</p>{' '}
-                      <p className="value">{entry.value}</p>
-                    </span>
-                  ))}
+                  {rawModal.headerProperties.groups.map((group, idx) =>
+                    group.entries.map((entry, idy) => (
+                      <span key={`${idx}_${idy}`} className="optional-name">
+                        <p className="caption">{entry.caption}:</p>{' '}
+                        <p className="value">{entry.value}</p>
+                      </span>
+                    ))
+                  )}
```

Comment:
@siemiatj  in case of raw modals, 
why not  adding a group separator (e.g. `<span key={`${idx}_${cursor}`} className="separator" />`) after each group?

Reply:
Because it'll look stupid if there's a separator dangling at the end without anything following.